client/client_test.go

```diff
@@ -147,16 +147,17 @@ func TestCanPerformStartTLS(t *testing.T) {
 			wantErr:       false,
 		},
 	}
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
+	for _, scenario := range scenarios {
+		item := scenario
```

Comment:
I don't really understand what this is changing here: the `item` variable isn't necessary, you can just use `scenario` directly 🤔